README.md

```diff
@@ -5,4 +5,4 @@ Where we will show off git
 Three goals we want to show:
 * showing a commit briefly
 * updating changes (so everybody else has them)
-* something else
\ No newline at end of file
+* pulling in changes from a remote
```

Comment:
This looks great!  A couple of suggested changes:

- [ ] Update wording to include command line (or terminal)
- [ ] Add more detail to the third bullet?